cypress/integration/dashboard_spec.js

```diff
@@ -1,3 +1,40 @@
 describe('The Dashboard Page', function () {
+    beforeEach(() => {
+        cy.request({
```

Comment:
With e2e tests I thought we were supposed to interact with the UI rather than send requests ourselves?

Reply:
Hmm, yeah I guess so, I'll change this now